files/build_templates/sonic_debian_extension.j2

```diff
@@ -323,6 +323,10 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-db-cli_*.deb || \
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/sonic-rsyslog-plugin_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Generate host conf for rsyslog_plugin
```

Comment:
You can create this file directly into $FILESYSTEM_ROOT_ETC/rsyslog.d/host_events.conf

Reply:
Tried to do this, but run into permission issues

Reply:
You may have to run the j2 as sudo
It is ok, either way

Reply:
Tried as sudo but still same error, not sure